softlayer/resource_softlayer_virtual_guest.go

```diff
@@ -719,6 +720,7 @@ func resourceSoftLayerVirtualGuestRead(d *schema.ResourceData, meta interface{})
 	d.Set("private_network_only", *result.PrivateNetworkOnlyFlag)
 	d.Set("hourly_billing", *result.HourlyBillingFlag)
 	d.Set("local_disk", *result.LocalDiskFlag)
+	d.Set("post_install_script_uri", *result.PostInstallScriptUri)
```

Comment:
SL API does not support PostInstallScriptUri. See the link : https://stackoverflow.com/questions/44728911/how-to-get-postinstallscripturi-information-of-softlayer-vitual-guest